src/hotspot/share/runtime/thread.hpp

```diff
@@ -621,11 +621,6 @@ class Thread: public ThreadShadow {
 
   bool is_trace_suspend()               { return (_suspend_flags & _trace_flag) != 0; }
 
-  // VM operation support
-  int vm_operation_ticket()                      { return ++_vm_operation_started_count; }
-  int vm_operation_completed_count()             { return _vm_operation_completed_count; }
-  void increment_vm_operation_completed_count()  { _vm_operation_completed_count++; }
```

Comment:
Ditto for `_vm_operation_completed_count`?

Reply:
Fixing